package.json

```diff
@@ -35,6 +35,7 @@
         "helmet": "^3.21.2",
         "i18next-browser-languagedetector": "^4.0.2",
         "i18next-http-backend": "^1.0.3",
+        "icomoon-react": "^2.0.16",
```

Comment:
I don't love having this dependency. I'm creating a ticket to work on next week to turn these into regular SVGs. https://github.com/cagov/benefits-screener/issues/50